lib/rules/comment-whitespace-inside/index.mjs

```diff
@@ -65,44 +65,53 @@ const rule = (primary, _secondaryOptions, context) => {
 				return;
 			}
 
-			const leftMatches = rawComment.match(/(^\/\*+)(\s)?/);
+			const leftMatches = rawComment.match(/(^\/\*+)(\s*)/);
 
 			if (leftMatches == null || leftMatches[1] == null) {
 				throw new Error(`Invalid comment: "${rawComment}"`);
 			}
 
-			const rightMatches = rawComment.match(/(\s)?(\*+\/)$/);
+			const rightMatches = rawComment.match(/(\s*)(\*+\/)$/);
 
 			if (rightMatches == null || rightMatches[2] == null) {
 				throw new Error(`Invalid comment: "${rawComment}"`);
 			}
 
 			const opener = leftMatches[1];
-			const leftSpace = leftMatches[2] || '';
-			const rightSpace = rightMatches[1] || '';
+			const leftWord = leftMatches[2] || '';
+			const leftSpace = leftWord[0] || '';
+			const rightWord = rightMatches[1] || '';
+			const rightSpace = rightWord[0] || '';
```

Comment:
[suggest] This is not required, but the following would be a bit more readable for me:

- Use `??` instead of `||`.
- Use `charAt()`, which always returns a string.

```suggestion
			const leftWord = leftMatches[2] ?? '';
			const leftSpace = leftWord.charAt(0);
			const rightWord = rightMatches[1] ?? '';
			const rightSpace = rightWord.charAt(0);
```